mappings/net/minecraft/structure/piece/StructurePieceType.mapping

```diff
@@ -67,6 +67,9 @@ CLASS net/minecraft/unmapped/C_etqvykma net/minecraft/structure/piece/StructureP
 		ARG 0 simplePieceType
 		ARG 1 id
 	CLASS C_blzadryc ManagerAware
+		METHOD load load (Lnet/minecraft/unmapped/C_zxhjfyir;Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
+			ARG 1 structureTemplateManager
+			ARG 2 nbt
 	CLASS C_cdoppjru Simple
-		METHOD load (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
+		METHOD load load (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_coopoqiy;
```

Comment:
hm...?

Reply:
just to show that it's mapped :p

Reply:
oh, i'm pretty sure that Enigma does have a `Mark as Unobfuscated` feature, but yeah, maybe it does exactly that...?